datafusion/execution/src/cache/cache_manager.rs

```diff
@@ -42,8 +47,18 @@ pub type FileStatisticsCache =
 /// especially when done over remote object stores.
 ///
 /// See [`crate::runtime_env::RuntimeEnv`] for more details
```

Comment:
This is technically a breaking API change (a good one in my mind). I am just pointing it out